catalog/networking/network/README.md

```diff
@@ -43,6 +43,8 @@ region                              9
 source-subnetwork-ip-ranges-to-nat  1
 vpn-secret-key                      1
 vpn-secret-name                     1
+vpn-tunnel-peer-ip-01               1
+vpn-tunnel-peer-ip-02               1
```

Comment:
nit: further down this readme there is a [resources](https://github.com/GoogleCloudPlatform/blueprints/tree/main/catalog/networking/network#resources) section which also needs to be [updated](https://github.com/GoogleCloudPlatform/blueprints/blob/main/utils/fix-readmes.sh)

Reply:
Acked.